pyproject.toml

```diff
@@ -34,8 +34,7 @@ classifiers = [
 requires-python = ">=3.8"
 
 dependencies = [
-    "hdx-python-scraper@git+ssh://git@github.com/OCHA-DAP/hdx-python-scraper@hapi_metadata#egg=hdx-python-scraper",
-    "hdx-python-country@git+ssh://git@github.com/OCHA-DAP/hdx-python-country@admin_dataset#egg=hdx-python-country",
```

Comment:
Should we keep hdx-python-country as a dependency, since it's imported directly in `utilities/locations.py`? I know that it's not strictly needed because it's a dependency of the scraper, but I thought it's good practice to include all explicitly imported packages as dependencies, just in case e.g. relative dependencies change or something like that. Same for the HXL package -- I know we started having this discussion somewhere else but I don't think we ever wrapped it up.

Reply:
I can see that depending on the dependencies specified in third party libraries for direct dependenices could lead to trouble and so it is better to declare them - something that needs checking I think. 

In the case of the HDX libraries, I upgrade them all together, so there is less need, although if we start specifying those direct dependencies of HDX library subdependencies, it would save me from upgrading multiple packages at once (ie. when I change HDX Python Utilities, I upgrade Country, API and Scraper with new requirements to force any user of the parent library to get the latest versions of the child ones).

I think you're right - I'll add the country dependency.